src/middlewares/parkingValidation.js

```diff
@@ -0,0 +1,42 @@
+import Joi from '@hapi/joi';
+
+class ParkingValidation {
+  static validateParking(req, res, next) {
+    const schema = Joi.object({
+      name: Joi.string()
+        .min(3)
```

Comment:
can you add .trim because we don't need to save a name with extra spaces

Reply:
when you use trim only, there will still be white spaces so add trim().strict(). that should care of the white space problem

Reply:
it's been resolved